R/build-reference.R

```diff
@@ -291,7 +291,7 @@ data_reference_topic <- function(topic,
                                  run_dont_run = FALSE
                                  ) {
   local_context_eval(pkg$figures, pkg$src_path)
-  withr::local_options(list(downlit.rdname = topic$name))
+  withr::local_options(list(downlit.rdname = fs::path_ext_remove(topic$file_out)))
```

Comment:
Should we make this into a helper function? (Since we got it wrong in two places)